modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -280,6 +284,12 @@ public void createUser(SAMLConfig config) {
         }
     }
 
+    public void createApplicationForSAMLAartResolve(SAMLConfig config, String appName) throws Exception {
+
+        samlArtResolve =true;
```

Comment:
Fix formatting.

Reply:
Fixed https://github.com/wso2/product-is/pull/8590/commits/db6519d17a39bc68aa08982b911eb1fe830fe1a9